src/EmbeddedVsix/source.extension.vsixmanifest

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<PackageManifest Version="2.0.0" xmlns="http://schemas.microsoft.com/developer/vsx-schema/2011" xmlns:d="http://schemas.microsoft.com/developer/vsx-schema-design/2011">
+    <Metadata>
+        <Identity Id="SonarLint.AdditionalFiles.95521f6c-c3e6-460d-b0e6-aa64acd2fdb1" Version="1.0" Language="en-US" Publisher="SonarSource" />
+        <DisplayName>SonarLint.AdditionalFiles</DisplayName>
+        <Description xml:space="preserve">VSIX to package additional files required by non-Roslyn analyzers</Description>
+    </Metadata>
+    <Installation>
+        <InstallationTarget Id="Microsoft.VisualStudio.Community" Version="[14.0, 17.0)" />
```

Comment:
This range spans VS2015-2019 to be on the safe side. I'm not sure what would happen at installation time if the embedded VSIX could not be installed but the outer VSIX could.